marklogic-data-hub-central/ui/src/config/tiles.config.ts

```diff
@@ -82,6 +88,19 @@ const tiles: Record<TileId, TileItem>  = {
         controlColor: '#777',
         controls: ['menu', 'close'],
         intro: 'Monitor Flows and Steps',
+        toolbarHidden: false
+    },
+    detail: {
+        title: 'Explore - Details',
+        iconType: 'custom',
+        icon: 'exploreIcon',
+        color: '#376F63',
+        bgColor: '#F4F6F8',
+        border: '#BFBFBF',
+        controlColor: '#777',
+        controls: ['menu', 'close'],
+        intro: 'Detail record view',
```

Comment:
I don't see this actually displayed for Details so let's keep the intro value empty, "", to avoid confusion?